rxandroid/src/main/AndroidManifest.xml

```diff
@@ -12,4 +12,4 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-<manifest package="rx.android" />
\ No newline at end of file
+<manifest package="io.reactivex.android" />
```

Comment:
they changed from `rx` to `io.reactivex` and hence didn't needed to add a 2 since package name is already different